tests/acceptance/track-page/view-track-test.js

```
@@ -12,118 +12,120 @@
 
   test('it renders for anonymous user', async function (assert) {
     testScenario(this.server);
-    createTrackLeaderboardEntries(this.server, 'go', 'redis');
+    createTrackLeaderboardEntries(this.server, 'rust', 'redis');
 
-    await visit('/tracks/go');
-    assert.strictEqual(1, 1); // dummy assertion
+    await visit('/tracks/rust');
+    await this.pauseTest();
```

ESLint

[error] Do not commit `pauseTest()`

(ember/no-pause-test)